p2p/syncer_test.go

```diff
@@ -49,10 +49,32 @@ func TestSyncer(t *testing.T) {
 	b := sim.Chain[len(sim.Chain)-1]
 	n1.s.Broadcast(&MsgRelayBlock{b})
 
-	time.Sleep(time.Second)
+	time.Sleep(time.Second / 2)
```

Comment:
prefer using a smaller unit instead of a fraction: `500 * time.Millisecond`

Reply:
fixed